vdbench.py

```diff
@@ -1,367 +1,367 @@
-# -*- coding: utf-8 -*-
-#pylint : disable=E0401
-"""
-@author: SUSHANT KEDAR				
-DATACORE SOFTWARE PVT LTD CONFIDENTIAL
-THIS SPEC IS THE PROPERTY OF DATACORE SOFTWARE PVT LTD.IT SHALL NOT BE
-COPIED, USED,TRANSLATED OR TRANSFERRED IN WHOLE OR IN PART TO ANY THIRD
-PARTY WITHOUT PRIOR WRITTEN PERMISSION OF DATACORE SOFTWARE PVT LTD.
-File Name	:	vdbench.py
-Description	:	This script used to execute vdbench tool and create
-                HTML reports.
-
-"""
-import os
-import sys
-from configparser import ConfigParser
-import subprocess
-path_ = os.path.abspath("../../../Interface/REST")
-sys.path.insert(0, path_)
-from ILDC import ILDC
-sys.path.insert(0, os.path.abspath("../../../Lib/VdBenchLib"))
-from error_log import LogCreat
-
-class VdBenchRun():
-    '''
-    Class:- VdBenchRun
-    This class executes vdbench 4k fill and VSI/VDI/ORACLE/SQL
-    wokload as user request.
-    Arguments : None
-    Return: None
-    '''
-    def __init__(self):
-        configur = ConfigParser()
-        configur.read(r"../../../Config/VdBench_config/VDBench_config.ini")
```

Comment:
The File Path should not be hardcoded. Global Variable must be used to store paths or filenames

Reply:
,Global variables should be inside a module is a good practice, bad practice is that these are reference outside the module